templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete car</h1>
+  <p>Are you sure you want to delete car: {{ car }}</p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger">
+    <a href="{% url 'taxi:car-detail' pk=car.id %}"><button type="button" class="btn btn-success">Cancel</button></a>
```

Comment:
Looks a bit uncommon that `yes, delete` button is red and `cancel` is green. You can make it blue or smth

Reply:
Even class called `btn_success` for cancel, which is not clear